### PR TITLE
Fix -spec as per intended usage

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Comment by lrascao** [comment]
👍

File: include/jobs.hrl [hunk 1]
```diff
@@ -33,7 +33,9 @@
 		    | {config, file:name()}
                     | {group_rates, [{q_name(), [option()]}]}
 		    | {counters,    [{q_name(), [option()]}]}
-		    | {interval, integer()}.
+		    | {interval, integer()}
+            | {max_time, non_neg_integer()}
+            | {max_size, non_neg_integer()}.
```
Comment (uwiger) [inline include/jobs.hrl]: Both `max_time` and `max_size` can actually also be `undefined`, which is interpreted as "no upper limit".

**Comment by uwiger** [comment]
The implementation actually explicitly allows setting `max_time` and `max_size` to `undefined`, which are also the default values. This means "no limit". The documentation does not reflect this, however, which should be fixed.

**Comment by paulo-ferraz-oliveira** [comment]
I updated the PR branch

**Comment by uwiger** [comment]
Excellent - thanks!

**Pull request merged**